variables.tf

```diff
@@ -199,6 +199,18 @@ variable "primary_metastore_port" {
   default     = "9083"
 }
 
+variable "primary_metastore_host_read_only" {
+  description = "Optional Primary Hive Metastore READ ONLY hostname configured in Waggle Dance."
```

Comment:
`Primary Hive Metastore READ ONLY hostname configured in Waggle Dance. Optional.` reads better.